storybook/src/stories/UseResizeObserver/UseResizeObserver.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Canvas, Meta, Source, Story } from '@storybook/addon-docs'
+import * as stories from './stories'
+import TYPES from '../../../../too-many-hooks/dist/useResizeObserver.d.ts?raw'
+
+<Meta title="Example/UseResizeObserver" />
+
+# useResizeObserver
+
+Handles adding and removing a `ResizeObserver` to observe an element and return the most recent `ResizeObserverEntry`
+
+## Types
+
+<Source language="ts" code={TYPES} />
+
+## Examples
+
+### Container Query
```

Comment:
Is this named "Container Query" because it mimics a CSS container query?

I'd say it's different enough that we can name it something specific to what it's doing, namely that we're using the resize observer to alter the flex direction (using CSS-in-JS logic). So maybe "Flex Direction"?

Reply:
Renamed to "Container Query Emulation" per offline discussion